src/test/java/io/reactivex/internal/operators/flowable/FlowableMapTest.java

```diff
@@ -256,139 +256,13 @@ public Integer apply(Integer i) {
         }).blockingGet();
     }
 
-    // FIXME RS subscribers can't throw
-//    @Test(expected = OnErrorNotImplementedException.class)
-//    public void verifyExceptionIsThrownIfThereIsNoExceptionHandler() {
-//
-//        Publisher<Object> creator = new Publisher<Object>() {
-//
-//            @Override
-//            public void subscribe(Subscriber<? super Object> observer) {
-//                observer.onSubscribe(EmptySubscription.INSTANCE);
-//                observer.onNext("a");
-//                observer.onNext("b");
-//                observer.onNext("c");
-//                observer.onComplete();
-//            }
-//        };
-//
-//        Function<Object, Flowable<Object>> manyMapper = new Function<Object, Flowable<Object>>() {
-//
-//            @Override
-//            public Flowable<Object> apply(Object object) {
-//                return Flowable.just(object);
-//            }
-//        };
-//
-//        Function<Object, Object> mapper = new Function<Object, Object>() {
-//            private int count = 0;
-//
-//            @Override
-//            public Object apply(Object object) {
-//                ++count;
-//                if (count > 2) {
-//                    throw new RuntimeException();
-//                }
-//                return object;
-//            }
-//        };
-//
-//        Consumer<Object> onNext = new Consumer<Object>() {
-//
-//            @Override
-//            public void accept(Object object) {
-//                System.out.println(object.toString());
-//            }
-//        };
-//
-//        try {
-//            Flowable.create(creator).flatMap(manyMapper).map(mapper).subscribe(onNext);
-//        } catch (RuntimeException e) {
-//            e.printStackTrace();
-//            throw e;
-//        }
-//    }
-
     private static Map<String, String> getMap(String prefix) {
         Map<String, String> m = new HashMap<String, String>();
         m.put("firstName", prefix + "First");
         m.put("lastName", prefix + "Last");
         return m;
     }
 
-    @Test//(expected = OnErrorNotImplementedException.class)
-    @Ignore("RS subscribers can't throw")
-    public void testShouldNotSwallowOnErrorNotImplementedException() {
```

Comment:
Please don't remove ignored test methods as they fulfill the crucial task of not getting killed on CI due to lack of test output.